lib/rules/arrow-parens.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @fileoverview Rule to require parens in arrow function arguments.
+ * @author Jxck
```

Comment:
Without your real name, the copyright is meaningless. Up to you whether or not that matters.